spec/controllers/queue_items_controller_spec.rb

```diff
@@ -123,30 +123,14 @@
   describe "POST update_queue" do
     it_behaves_like "requires sign in" do
       let(:action) do
-        post :update_queue, queue_items: [
-          { id: 2, position: 3 }, { id: 1, position: 1 }
-        ]
+        post :update_queue, queue_items: [{id: 2,  position: 3}, {id: 1, position: 1}]
```

Comment:
Line is too long. [86/80]<br>Unnecessary spacing detected.<br>Space inside { missing.<br>Space inside } missing.